src/Uno.UI/Microsoft/UI/Xaml/Controls/PipsPager/PipsPagerTemplateSettings.Properties.cs

```diff
@@ -0,0 +1,17 @@
+using System.Collections.Generic;
+using Windows.UI.Xaml;
+
+namespace Microsoft.UI.Xaml.Controls
+{
+	public partial class PipsPagerTemplateSettings : DependencyObject
```

Comment:
Looks like there is no accessible constructor for this, but again, WinUI sources indicates there is a public constructor.

Not sure what's going on..